src/components/display-card/helper/csv_helper.js

```diff
@@ -0,0 +1,26 @@
+const csvHelper = (records) => {
+    const headers = [
+        "Id",
+        "Original",
+        "Reference",
+        "HumanTranslation",
+        "NeuroTranslation",
+        "StatyTranslation",
+        "PlayerChoice"
+    ];
+    const parsedArr = records.map(
+        (card, index) => {
+            const id = [`${index + 1}`]
+            const cardArr = Object.values(card)
+            const originToHuman = cardArr.slice(0, 3)
```

Comment:
I guess there is some problem here. When I tried to download the file, the format is not consistent with what I expect. Can you please check? I am attaching the file below.

[5237d10d-9519-4fa0-a86b-14ceafd21d85.csv](https://github.com/The-Brainiacs-FSE/TuringTestGame/files/9999529/5237d10d-9519-4fa0-a86b-14ceafd21d85.csv)

Reply:
Can you reproduce? I tried to reproduce but did not find the bug.

[b4ec6eec-5ff0-4ce5-b9eb-4e0f3172a93e.csv](https://github.com/The-Brainiacs-FSE/TuringTestGame/files/10000252/b4ec6eec-5ff0-4ce5-b9eb-4e0f3172a93e.csv)